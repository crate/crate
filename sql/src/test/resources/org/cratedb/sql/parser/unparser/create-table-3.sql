CREATE TABLE t1 (
col1 INTEGER NOT NULL,
col2 INTEGER,
col3 smallint NOT NULL, 
col4 smallint,
col5 bigint NOT NULL, 
col6 bigint,

col10 CHAR(1) NOT NULL, 
col11 CHAR(1),
col12 VARCHAR(1) NOT NULL,
col13 VARCHAR(1),
col14 LONG VARCHAR NOT NULL, 
col15 LONG VARCHAR,

col20 FLOAT NOT NULL, 
col21 FLOAT,
col22 REAL NOT NULL,
col23 REAL,
col24 DOUBLE NOT NULL, 
col25 DOUBLE, 

col30 DATE NOT NULL,
col31 DATE, 
col32 TIME NOT NULL,
col33 time, 
col34 timestamp NOT NULL,
col35 timestamp,

col40 CLOB NOT NULL, 
col41 CLOB, 
col42 BLOB NOT NULL,
col43 BLOB,

col50 DECIMAL NOT NULL, 
col51 DECIMAL, 
col52 DECIMAL (1) NOT NULL, 
col53 DECIMAL (1) ,
col54 DECIMAL (10) NOT NULL,
col55 DECIMAL (10),
col56 DECIMAL (1,1) NOT NULL,
col57 DECIMAL (1,1),
col58 DECIMAL (10,1) NOT NULL,
col59 DECIMAL (10,1),
col60 DECIMAL (10,10) NOT NULL,
col61 DECIMAL (10,10),
col62 DECIMAL (30,10) NOT NULL,
col63 DECIMAL (30,10),

col70 NUMERIC NOT NULL,
col71 NUMERIC
)