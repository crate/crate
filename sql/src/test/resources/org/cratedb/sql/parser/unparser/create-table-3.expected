CREATE TABLE t1(col1 INTEGER NOT NULL, col2 INTEGER, col3 SMALLINT NOT NULL, col4 SMALLINT, col5 BIGINT NOT NULL, col6 BIGINT, col10 CHAR(1) NOT NULL, col11 CHAR(1), col12 VARCHAR(1) NOT NULL, col13 VARCHAR(1), col14 LONG VARCHAR NOT NULL, col15 LONG VARCHAR, col20 DOUBLE NOT NULL, col21 DOUBLE, col22 REAL NOT NULL, col23 REAL, col24 DOUBLE NOT NULL, col25 DOUBLE, col30 DATE NOT NULL, col31 DATE, col32 TIME NOT NULL, col33 TIME, col34 TIMESTAMP NOT NULL, col35 TIMESTAMP, col40 CLOB(2147483647) NOT NULL, col41 CLOB(2147483647), col42 BLOB(2147483647) NOT NULL, col43 BLOB(2147483647), col50 DECIMAL(5,0) NOT NULL, col51 DECIMAL(5,0), col52 DECIMAL(1,0) NOT NULL, col53 DECIMAL(1,0), col54 DECIMAL(10,0) NOT NULL, col55 DECIMAL(10,0), col56 DECIMAL(1,1) NOT NULL, col57 DECIMAL(1,1), col58 DECIMAL(10,1) NOT NULL, col59 DECIMAL(10,1), col60 DECIMAL(10,10) NOT NULL, col61 DECIMAL(10,10), col62 DECIMAL(30,10) NOT NULL, col63 DECIMAL(30,10), col70 NUMERIC NOT NULL, col71 NUMERIC)